openpdf/src/main/java/com/lowagie/text/VersionBean.java

```diff
@@ -53,7 +55,9 @@ void setManifest(Manifest manifest) {
      *
      * @author asmirnov@exadel.com (latest modification by $Author$)
      * @version $Revision$ $Date$
+     * @deprecated simce 1.3.11. Use Document.RELEASE
```

Comment:
Use `@Deprecated(since = "1.3.11", forRemoval =...)` instead of specifying version here (or in conjunction with, but then fix typo). And do the same with other added deprecations